llvm/lib/Target/AMDGPU/VOPInstructions.td

```diff
@@ -1705,4 +1933,4 @@ def VOPTrue16Table : GenericTable {
 
   let PrimaryKey = ["Opcode"];
   let PrimaryKeyName = "getTrue16OpcodeHelper";
-}
```

Comment:
Remove whitespace change

Reply:
Hmmm I don't think I change anything here. Not sure why there is a diff

Reply:
Perhaps clang-format? It seems to have deleted the blank line. I don't see a clear answer on whether a EoF blank line is required for tablegen, but would prefer to keep it.